src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -2796,6 +2790,37 @@ public void Copy()
                 }
             }
 
+            /// <summary>Implements <see cref="IInteractiveWindowOperations2.CopyInputs"/>.</summary>
+            public void CopyInputs()
+            {
+                NormalizedSnapshotSpanCollection spans;
+                if (TextView.Selection.IsEmpty)
+                {
+                    spans = GetSpansFromCurrentLine();
+                }
+                else
+                {
+                    spans = GetSpansFromSelectedLines();
```

Comment:
@amcasey suggests we stick with the behavior of regular copy here, i.e. only copy from selected text. I will change this unless someone has a better argument :)